packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -302,7 +302,7 @@ class HierarchicalTermSelector extends Component {
 
 			// If the term's name contains the filterValue, or it has children
 			// (i.e. some child matched at some point in the tree) then return it.
-			if ( -1 !== term.name.toLowerCase().indexOf( filterValue ) || term.children.length > 0 ) {
+			if ( -1 !== term.name.toLowerCase().indexOf( filterValue.toLowerCase( ) ) || term.children.length > 0 ) {
```

Comment:
Nitpick: There should be no spaces between the parentheses here.

>No filler spaces in empty constructs (e.g., {}, [], fn()).

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

```suggestion
			if ( -1 !== term.name.toLowerCase().indexOf( filterValue.toLowerCase() ) || term.children.length > 0 ) {
```

Reply:
Yes @Jackie6, just remove that or commit the suggestion and we're good to go!